apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -0,0 +1,510 @@
+        description={
+          <>
+            Restore to new project is only available for Postgres 15 and above.
+            <br /> Go to{' '}
+            <Link className="underline" href={`/project/${project?.ref}/settings/infrastructure`}>
+              infrastructure settings
+            </Link>{' '}
+            to upgrade your database version.
+          </>
+        }
+      />
+    )
+  }
+
+  if (!PHYSICAL_BACKUPS_ENABLED) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project requires physical backups"
+        description={
+          <>
+            Enable physical backups to restore to a new project.
+            <br /> Find out more about how backups work at supabase{' '}
+            <Link
+              target="_blank"
+              className="underline"
+              href="https://supabase.com/docs/guides/platform/backups"
+            >
+              in our docs
+            </Link>
+          </>
+        }
+      />
+    )
+  }
+
+  if (plan === 'free') {
+    return (
+      <UpgradeToPro
+        buttonText="Upgrade"
+        primaryText="Restore to a new project requires a pro plan or above."
+        secondaryText={
+          'To restore to a new project, you need to upgrade to a Pro plan and have physical backups enabled.'
+        }
+      />
+    )
+  }
+
+  if (isError) {
+    return <AlertError error={error} subject="Failed to retrieve backups" />
+  }
+
+  if (!isActiveHealthy) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project is not available while project is offline"
+        description={<>Your project needs to be online to restore your database to a new project</>}
+      />
+    )
+  }
+
+  if (cloneStatus?.status === 'FAILED') {
+    return (
+      <Admonition
+        type="destructive"
+        title="Failed to restore to new project"
+        description={
+          <>
+            The new project failed to be created.
+            <br />
+            <Link className="underline" href={`/support/new?category=dashboard_bug`}>
+              Contact support
+            </Link>
+          </>
+        }
+      />
+    )
+  }
+
+  if (cloneStatus?.status === 'COMPLETED') {
+    return (
+      <Admonition
+        type="default"
+        title="Restoration completed"
+        description={
+          <>
+            The new project has been created.
+            <br />
+            <Link className="underline" href={`/project/${cloneStatus?.target.ref}`}>
+              Go to new project
+            </Link>
+          </>
+        }
+      />
+    )
+  }
+
+  if (cloneStatus?.status === 'IN_PROGRESS') {
+    return (
+      <Alert_Shadcn_ className="[&>svg]:bg-none! [&>svg]:text-foreground-light">
+        <Loader2 className="animate-spin" />
+        <AlertTitle_Shadcn_>Restoration in progress</AlertTitle_Shadcn_>
+        <AlertDescription_Shadcn_>
+          The new project is being created.
```

Comment:
if the new project name is available here
`Your project is currently being restored to {newProject.name}`